src/kiota/KiotaHost.cs

```diff
@@ -541,6 +545,14 @@ private static Option<bool> GetClearCacheOption(bool defaultValue)
         clearCacheOption.AddAlias("--cc");
         return clearCacheOption;
     }
+
+    private static Option<bool> GetDisableSSLValidationOption(bool defaultValue)
+    {
```

Comment:
I'm not sure we'll want this as a new option on the CLI since it's a fairly specific use case.
Plus it's likely that a person disabling that, will want it disabled for all commands being run.
Question for you and @andrueastman was using an environment variable instead considered?

Reply:
I haven't considered using an environment variable, after going through the ticket first thing came into my mind was to add a new option on the CLI. But In my views setting it in the CLI will allow developer the flexibility of turning it on and off frequently. I can do the change for env variable if you like. Thanks :)

Reply:
Thanks for the additional information
@sebastienlevert do you have a preference here?

Reply:
> Plus it's likely that a person disabling that, will want it disabled for all commands being run.
+1

Whichever way we go, we'll also probably need a PR to this file as well with an update. https://github.com/microsoftdocs/openapi-docs/blob/main/OpenAPI/kiota/using.md

Reply:
> Whichever way we go, we'll also probably need a PR to this file as well with an update. https://github.com/microsoftdocs/openapi-docs/blob/main/OpenAPI/kiota/using.md
Hi @andrueastman Should I create another issue and a new PR for that or it's fine if I do that in this issue itself?

Reply:
before we make any further changes, I'd like to get the input from Seb.

Reply:
an option on the command was the discussion resolution. Thank you for your patience. Let's start a documentation PR as well.